src/cirrus/management/commands/manage.py

```diff
@@ -314,3 +314,126 @@ def get_payloads(
         rerun,
     ):
         click.echo(json.dumps(payload, default=str))
+
+
+@manage.command("list-workflows")
+@pass_deployment
+def list_workflows(deployment: Deployment):
+    """List available collections/workflows combinations"""
+    workflows = deployment.get_workflows()
+    click.echo(json.dumps(workflows, indent=2))
+
+
+@manage.command("get-workflow-summary")
+@click.argument("collections")
+@click.argument("workflow_name")
+@click.option(
+    "--since",
+    default=None,
+    help=(
+        "Only include items updated since this relative duration (e.g., 7d, 36h, 15m)"
+    ),
+)
+@click.option(
+    "--limit",
+    default=10000,
+    show_default=True,
+    type=int,
```

Comment:
Looking for the above link I found that click has its own int type, `click.INT`. I'm not sure what the difference is in using that vs `int`, but it must have it for a reason: https://click.palletsprojects.com/en/stable/parameter-types/#built-in-types-listing.

Reply:
Vetted all the types and made updates where appropriate.